### ISSUE TITLE
docker compose up error

### ISSUE THREAD

**Issue by Galaxyjia** [strapi/strapi-docker#290]
![屏幕快照 2021-03-30 下午11 40 58](https://user-images.githubusercontent.com/40594113/113016892-7dc16f80-91b1-11eb-8709-d69e7181429e.png)

**Comment by mattkaye** [comment]
Yup, I just followed the directions straight from the docs and got the same error but complaining about a different file

<img width="708" alt="image" src="https://user-images.githubusercontent.com/2780089/114304613-2620df00-9aa2-11eb-805c-5c081253501e.png">
